src/win/internal.h

```diff
@@ -302,6 +302,7 @@ int uv_translate_sys_error(int sys_errno);
  * FS
  */
 void uv_fs_init();
+typedef int (*uv__fs_mktemp_cb)(uv_fs_t* req, static const size_t num_x, WCHAR *ep);
```

Comment:
can we call this `uv__fs_mktemp_func`? I don't like using "callback" here, since we already have a callback for fs requests which is something else.